libs/shared/src/types/environment/index.ts

```diff
@@ -1,9 +1 @@
 export type EnvironmentId = string;
-
-export enum ApiRateLimitCategoryTypeEnum {
-  TRIGGER = 'trigger',
-  CONFIGURATION = 'configuration',
-  GLOBAL = 'global',
-}
-
-export type IApiRateLimits = Record<ApiRateLimitCategoryTypeEnum, number>;
```

Comment:
Refactoring this typing from the Environment repository due to the fact that the categories are specific to rate limiting configuration, therefore belong in the Rate Limit typings.